music_assistant/providers/ytmusic/__init__.py

```diff
@@ -588,6 +592,44 @@ async def get_stream_details(self, item_id: str, media_type: MediaType) -> Strea
             stream_details.audio_format.sample_rate = int(stream_format.get("asr"))
         return stream_details
 
+    async def recommendations(self) -> list[RecommendationFolder]:
```

Comment:
```suggestion
    @use_cache(3600)
    async def recommendations(self) -> list[RecommendationFolder]:
```

and import use_cache from the cache module